site/it/about/bib.md

```diff
@@ -0,0 +1,131 @@
+ricerca farmaceutica computazionale. Questo documento descrive
+l'interfaccia di TensorFlow e l'implementazione dell'interfaccia
+che abbiamo realizzato in Google. Le API di TensorFlow e l'
+implementazione di riferimento sono state rilasciate come pacchetto 
+open-source con licenza Apache 2.0 nel Novembre 2015, e sono 
+disponibili su www.tensorflow.org.
+
+
+### In formato BibTeX
+
+Se usate TensorFlow nella vostra ricerca e volete citare il sistema TensorFlow, 
+vi suggeriamo di citare questo White Paper.
+
+<pre>
+@misc{tensorflow2015-whitepaper,
+title={ {TensorFlow}: Large-Scale Machine Learning on Heterogeneous Systems},
+url={https://www.tensorflow.org/},
+note={Software available from tensorflow.org},
+author={
+    Mart\'{\i}n~Abadi and
+    Ashish~Agarwal and
+    Paul~Barham and
+    Eugene~Brevdo and
+    Zhifeng~Chen and
+    Craig~Citro and
+    Greg~S.~Corrado and
+    Andy~Davis and
+    Jeffrey~Dean and
+    Matthieu~Devin and
+    Sanjay~Ghemawat and
+    Ian~Goodfellow and
+    Andrew~Harp and
+    Geoffrey~Irving and
+    Michael~Isard and
+    Yangqing Jia and
+    Rafal~Jozefowicz and
+    Lukasz~Kaiser and
+    Manjunath~Kudlur and
+    Josh~Levenberg and
+    Dandelion~Man\'{e} and
+    Rajat~Monga and
+    Sherry~Moore and
+    Derek~Murray and
+    Chris~Olah and
+    Mike~Schuster and
+    Jonathon~Shlens and
+    Benoit~Steiner and
+    Ilya~Sutskever and
+    Kunal~Talwar and
+    Paul~Tucker and
+    Vincent~Vanhoucke and
+    Vijay~Vasudevan and
+    Fernanda~Vi\'{e}gas and
+    Oriol~Vinyals and
+    Pete~Warden and
+    Martin~Wattenberg and
+    Martin~Wicke and
+    Yuan~Yu and
+    Xiaoqiang~Zheng},
+  year={2015},
+}
+</pre>
+
+O in forma testuale:
+
+<pre>
+Martín Abadi, Ashish Agarwal, Paul Barham, Eugene Brevdo,
+Zhifeng Chen, Craig Citro, Greg S. Corrado, Andy Davis,
+Jeffrey Dean, Matthieu Devin, Sanjay Ghemawat, Ian Goodfellow,
+Andrew Harp, Geoffrey Irving, Michael Isard, Rafal Jozefowicz, Yangqing Jia,
+Lukasz Kaiser, Manjunath Kudlur, Josh Levenberg, Dan Mané, Mike Schuster,
+Rajat Monga, Sherry Moore, Derek Murray, Chris Olah, Jonathon Shlens,
+Benoit Steiner, Ilya Sutskever, Kunal Talwar, Paul Tucker,
+Vincent Vanhoucke, Vijay Vasudevan, Fernanda Viégas,
+Oriol Vinyals, Pete Warden, Martin Wattenberg, Martin Wicke,
+Yuan Yu, and Xiaoqiang Zheng.
+TensorFlow: Large-scale machine learning on heterogeneous systems,
+2015. Software available from tensorflow.org.
+</pre>
+
+
+
+## TensorFlow: Un sistema per Machine Learning su Larga Scala
+
+[Accedi a questo white paper.](https://www.usenix.org/system/files/conference/osdi16/osdi16-abadi.pdf)
+
+**Sommario:** TensorFlow è un sistema di machine learning che
+opera su larga scala ed in ambienti eterogenei. TensorFlow
+usa grafi dataflow per rappresentare la computazione,
+stati condivisi, e le operazioni che cambiano questi stati.
+Esso mappa i nodi di un grafo dataflow tra molte macchine in cluster, 
```

Comment:
"Questo" for me sounds a bit better than "Esso". What do you think?

Reply:
I've no clear what translation style is the best for Google...: less literal and more focus on meaning (a better translation should be: "Tensorflow usa grafi dataflow per rappresentare la computazione, stati condivisi, e le operazioni che cambiano questi stati, mappando i nodi di un grafo dataflow tra molte macchine in cluster...");
or more literal and focused to respect the original language (and the best translation should be: "Tensorflow usa grafi dataflow per rappresentare la computazione, stati condivisi, e le operazioni che cambiano questi stati. Esso mappa i nodi di un grafo dataflow tra molte macchine in cluster..." because using "Questo" we suggest an "arbitral" causal effect from the first and second sentence. (If we use "Questo..." the overall meaning is that the use of graphs etc. produces as effect the mapping of nodes... Something is not in the original English sentence, I think...

Reply:
The problem here Is that Esso Is correct but It has a very low frequency in common use nowday. So It Is better "mappando" if the sentence will be not too long.

Reply:
I think It could be something like:
"Effettua la mappatura", "Si occupa di mappare", "Provvede a mappare" or any other workaround like these.

Reply:
> I've no clear what translation style is the best for Google...: less literal and more focus on meaning ... or more literal and focused to respect the original language

My personal opinion, if it helps ... as a large community docs project, the tensorflow.org tutorials and guides may have inconsistent style and grammar depending on doc origin. The most important thing is that the code snippets are correct and the technical details are preserved. I'm not too concerned if the original language is lost as long as the meaning remains.

As for style, tone, and formality, the (English) [Google dev docs style](https://developers.google.com/style/tone) says:
> Aim, in your documents, for a voice and tone that's conversational, friendly, and respectful without being overly colloquial or frivolous; a voice that's casual and natural and approachable, not pedantic or pushy. Try to sound like a knowledgeable friend who understands what the developer wants to do.

I don't know how that applies to Italian technical documentation. Maybe check some developer docs in other Google projects (like Firebase, Android, etc.) and see if you pick up any patterns (and blog posts tend to be much more informal).

And as you work on this project, you may very well find problems in the English source docs. Please, feel free to submit PRs there if you have a fix :)

Reply:
Thank you @lamberta for your addressing to Google dev doc style. Along the route I found a couple of small improvement to English source too; I'll push it as suggestions. :)